frontend/assets/stylesheets/components/_icons.scss

```diff
@@ -61,6 +61,12 @@
     width: 36px;
     height: 36px;
 }
+.icon-inline--badge {
+    width: 50px;
```

Comment:
Yeah, we have a task for it now https://github.com/guardian/membership-frontend/blob/master/frontend/Gruntfile.js#L71